packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -439,6 +465,38 @@ const PRESETS = [
             ['project.name']: 'mrt-reference-app'
         },
         private: true
+    },
+    {
+        id: 'app-sample-extension-local-install',
+        name: 'Application Extension (Local Install)',
```

Comment:
Is this up-to-date? This description suggests that an Application Extension project is going to be created, but based on the name of this PR you are generating a Project that uses Application extensibility.

Reply:
I've updated the naming. Let me know if it makes more sense now.